VivistaServer/Templates/base.liquid

```diff
@@ -48,10 +48,13 @@
 							<li class="pure-menu-item">
 								<a href="/settings" class="pure-menu-link">Settings</a>
 							</li>
-							{% if "Admin" == "Admin" %}
-								<li class="pure-menu-item">
-									<a href="/admin/dashboard" class="pure-menu-link">Dashboard</a>
-								</li>
+							{% if IsAdmin.Result %}

```

Comment:
Kun je het zo maken dat je niet hier nog de Result moet opvragen?

Reply:
Als ik die Result weg doe, en die IsAdmin opvraag. Dan doet die een toString() van het object waar ik niet mee verder kan.

Reply:
Ja, daarom moet je de Result in IsAdmin steken wanneer je die injecteert, in de HTMLRenderer.